### DISCUSSION TITLE
服务化部署PaddleOCR的QPS过低

### DISCUSSION THREAD

**Discussion by ouerum** [PaddlePaddle/PaddleOCR#13841] [Q&A] [answered]
按照官方教程服务化部署PaddleOCR-py，最终测试发现QPS只有4-5，距离官方指标60还有很大差距。想请教下还有没有优化的空间。下图是pipeline.tracer的截图，相较于其他指标，in的数值异常高，请问是什么原因？
<img width="786" alt="WeChatWorkScreenshot_2b15473b-c422-44b6-b47c-b6f1ad3c14c9" src="https://github.com/user-attachments/assets/faf4bb31-afe3-442a-b690-54792dd75af5">

**Comment by UserWangZz** [comment]
尝试循环跑100遍测试一个平均值看看

**Comment by ouerum** [comment, marked as answer]
应该是直接用了pipline_http_client.py的原因，后面改成用jmeter来做接口测试，QPS就有15+了。感谢回复！
